tests/test_git.py

```diff
@@ -0,0 +1,13 @@
+"""
+Unit-tests for git
+"""
+
```

Comment:
I like to attach imports directly to the module docstring. Of course, this is personal taste.